recipe/meta.yaml

```diff
@@ -18,8 +18,6 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - numpy  # [build_platform != target_platform]
-    - python  # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```

Comment:
Good catch! Dropped `cross-python` since it appears duplicated as well

Reply:
I hit the commit button too early when removing those. Thanks for catching the last one!

Reply:
Not at all. Missed these completely until I saw your change. Thanks for getting the ball rolling 🙂